packages/editor/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 ### Deprecations
 
 - `EditorGlobalKeyboardShortcuts` has been deprecated in favor of `VisualEditorGlobalKeyboardShortcuts`.
+- The `getAutosave`, `getAutosaveAttribute`, and `hasAutosave` selectors are deprecated. Please use the `getAutosave` selector in the `@wordpress/core-data` package.
```

Comment:
These notes are under a release which has already been published. They should be moved up to the notes for the pending unreleased version.